components/kserve/kserve_config_handler.go

```diff
@@ -109,7 +109,7 @@ func (k *Kserve) setDefaultDeploymentMode(ctx context.Context, cli client.Client
 			return fmt.Errorf("failed to list pods: %w", err)
 		}
 		for _, pod := range podList.Items {
-			pod := pod
+			pod := pod //nolint: copyloopvar
```

Comment:
```suggestion
```

An alternative to the 3 places that you've added the nolint to, could be to just delete those lines.

Reply:
right, thats better, let me fix it